### ISSUE TITLE
index.vue:41 Uncaught Error: TypeError: n2.zip.loadAsync is not a function

### ISSUE THREAD

**Issue by liwilljinx** [VolodymyrBaydalka/docxjs#42]
i use docx-preview in vue 3 . And this is my code
```
import * as docx from 'docx-preview'
import Docxtemplater from 'docxtemplater'
import JSZipUtils from 'jszip-utils'
import { saveAs } from 'file-saver'
import * as JSZip from 'jszip'

onMounted(() => {
  window.JSZip = JSZip
})

const exportWord = () => {
  JSZipUtils.getBinaryContent('template.docx', function(error, content) {
    console.log('-----', content)
    if (error) {
      throw error
    }
    const zip = new JSZip(content)
    console.log('+++++', zip)
    const doc = new Docxtemplater()
    console.log('/////', doc)
    doc.loadZip(zip)
    console.log('=====', doc)
    doc.setData({
      ...toRaw(props.row)
    })
    try {
      doc.render()
    } catch (error) {
      const e = {
        message: error.message,
        name: error.name,
        stack: error.stack,
        properties: error.properties
      }
      console.log(JSON.stringify({ error: e }))
      throw error
    }
    const out = doc.getZip().generate({
      type: 'blob',
      mimeType:
        'application/vnd.openxmlformats-officedocument.wordprocessingml.document'
    })
    previewShow.value = true
    docx.renderAsync(out, word)
    // saveAs(out, '测试.docx')
  })
}
```
how can i solve this problem.

**Comment by minamo7sen** [comment]
Try importing `jszip` first before `docx-preview`.

**Comment by VolodymyrBaydalka** [comment]
Are you sure that you use JSZip >=3.0.0? Since I don't have full code I can't tell how to solve your issue. Simple vue3 applicaton with docx-preview package works fine.

**Comment by liwilljinx** [comment]
> Are you sure that you use JSZip >=3.0.0? Since I don't have full code I can't tell how to solve your issue. Simple vue3 applicaton with docx-preview package works fine.

I try importing jszip first before docx-preview.And fix my problem, I use JSZip=3.10.0,thank you.
